### PR TITLE
Add milestone mpotane.json #2 👣.

### PR DESCRIPTION
**Pull request by mpotane**
Update 2 🎉

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4329"></a>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1 @@
-## 1.71.2 (2023-02-01)
-
-
-
+## 1.71.3 (2023-02-01)
```
Comment (Panquesito7) [inline CHANGELOG.md:1]:
```suggestion
## 1.71.2 (2023-02-01)


```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1 @@
 ## 1.71.2 (2023-02-01)
-
-
-
```
Comment (Panquesito7) [inline CHANGELOG.md:4]: These extra lines must be added in order for the file to not be modified.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (mpotane) [reply CHANGELOG.md]: Thanks

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,3 @@
 ## 1.71.2 (2023-02-01)
 
 
-
```
[the remark this answers is not in the document]
Comment (mpotane) [reply CHANGELOG.md]: Thanks 🙏

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Amazing. Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**